content/collections/experiment/en/notifications.md

```diff
@@ -0,0 +1,66 @@
+---
+id: e39f9287-bec8-4403-ba60-8b04da56b5f9
+blueprint: experiment
+title: Notifications through Slack Channel or Webhook
+this_article_will_help_you:
+  - 'Understand the type of notifications you can set for experiments'
+  - 'Create notification alerts for your experiments'
+landing: false
+exclude_from_sitemap: false
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1753390854
+---
+You can set up notification alerts for your experiments. Amplitude sends these notifications when your experiments transition status, when they start or end, or when major changes occur. There are two ways to receive notifications:
```

Comment:
we should make things clear which notifications are for experiments and which notifications are for both flags and experiments.